src/hexen/h2_main.c

```diff
@@ -1124,7 +1125,28 @@ static void CrispyDrawStats (void)
     right_widget_h = 0;
     right_widget_x = coord_x + WIDESCREENDELTA;
 
-    if (player->cheats & CF_SHOWFPS)
+    if (crispy->playercoords == WIDGETS_ALWAYS || (automapactive && crispy->playercoords == WIDGETS_AUTOMAP))
```

Comment:
I think we all used the `WIDGETS_ALWAYS` condition by accident, oops. Should be this for each game:
```c
if (automapactive && crispy->playercoords == WIDGETS_AUTOMAP)
```

Reply:
Why do you mean?

Reply:
I can set `crispy_playercoords 2` in the cfg which is a cheat according to this: https://github.com/fabiangreffrath/crispy-doom/wiki/Crispness#navigational

Reply:
It shouldn't be exposed in the menu, but it should still be possible to cheat it on the screen by editing the config file.